addons/website_livechat/tests/test_ui.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import tests, _
+from odoo.addons.website_livechat.tests.common import TestLivechatCommon
+
+
+@tests.tagged('post_install', '-at_install')
+class TestLivechatUI(tests.HttpCase, TestLivechatCommon):
+    def setUp(self):
```

Comment:
Same here, why not a setupClass ? (+ void line between class and first method, dbelinter)

Reply:
Seems that httpcase and savepointcase are complicated to mix, mostly for patching methods. But maybe I'm not expert enough in tests. :thinking: